docs/conda-env.yml

```
@@ -0,0 +1,19 @@
+name: compass-docs-env
+channels:
+  - conda-forge
+dependencies:
+  - python[version='>=3.9']
+  - numpy
+  - lxml
+  - gdal[version='>=3']
+  - pandas
+  - pyproj
+  - pysolid
+  - pytest
+  - raider
```

```suggestion
  - raider-base
```
IIRC we don't need all of `raider`

Should this also change in `COMPASS/requirements.txt`?

---

are we removing it entirely from `requirements.txt`?

---

Actually, I think it can be removed from the requirements. The only way we got it to work is to install `RAiDER` from source and install only the dependencies we need to run the troposphere delay. No need to install it as a `conda `dependency, in my opinion